doc/getting_started/installation_linux.rst

```diff
@@ -10,8 +10,10 @@ applications on the following Linux distributions:
 
 * Ubuntu 16.04 LTS 64-bit
 * Fedora 25 64-bit
+* Clear Linux
 
-Where needed, alternative instructions are listed for Ubuntu and Fedora.
+Where needed, alternative instructions are listed for Ubuntu, Fedora and
```

Comment:
There's another PR adding some archlinux tips, so maybe we should just rewrite this line here as:

    Where needed, alternative instructions are listed for specific Linux distributions.